release-notes/opendistro-for-elasticsearch-sql.release-notes-1.13.0.0.md

```diff
@@ -0,0 +1,57 @@
+## 20201-2-3 Version 1.13.0.0
+
+### Features
+* Added keywords option as alias identifier in SQL parser [#866] (https://github.com/opendistro-for-elasticsearch/sql/pull/866)
```

Comment:
writing it this way markdown will render url as text. could you follow the same format as previous release notes? example from 1.12 release notes:
```
* Add count() support for PPL ([#894](https://github.com/opendistro-for-elasticsearch/sql/pull/894))
```

Reply:
Ack, misread the formatting, will change